### DISCUSSION TITLE
Use specified interface

### DISCUSSION THREAD

**Discussion by bart3nder** [projectdiscovery/httpx#1206] [Ideas]
Hi! Hope you all doing well.
Thanks for the amazing tools, I have a small feature request :
Please add a feature to all pd tools which actively interact with target, Like **Nuclei**, HTTPx, DNSx, **Katana** and Naabu where the user can select an interface that tool send all traffic through it, Like the nmap `-e` flag (e.g: `-e <iface>: Use specified interface`). This is very useful when user have multiple network interface on the system and needs to pass e.g: Katana's crawling to `eth1`, Nuclei attack payloads to `eth2` and ...
Thanks!

**Comment by MetzinAround** [comment]
Thanks for the suggestion @bart3nder! Our team will take a look at this discussion when looking for new features to add.

**Comment by bart3nder** [comment]
Thanks!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
